src/app/Dashboard/components/LogPointsModal.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import TextField from '@material-ui/core/TextField';
+import StartIcon from '@material-ui/icons/Star';
+import { MenuItem, ButtonComponent } from '../../common/components';
+import actions from '../operations/actions';
+
+/**
+   * @name LogActivityForm
+   * @summary Returns Form
```

Comment:
The summery here does not explain what this component does?

Reply:
@Kachulio1 I have added a more descriptive summary